### PR TITLE
feat: faq, sponsor and footer section

### PR DESCRIPTION
**Pull request by uy-seng**

### CODE REVIEW

File: components/homeComponents/HomeFooter.tsx [hunk 1]
```diff
@@ -1,69 +1,96 @@
 import TwitterIcon from '@mui/icons-material/Twitter';
 import InstagramIcon from '@mui/icons-material/Instagram';
 import FacebookIcon from '@mui/icons-material/Facebook';
+import React, { SVGProps } from 'react';
 
+//TODO: need to do some media queries for footer, styling is off
```
Comment (DedsecKnight) [inline components/homeComponents/HomeFooter.tsx:6]: Is this something that needs to be addressed before merging this PR?

File: components/homeComponents/HomeFooter.tsx [hunk 2]
```diff
@@ -1,69 +1,96 @@
...
-          >
-            <InstagramIcon className="footerIcon" />
-          </a>
-          <a href="https://www.facebook.com/hackutd/" rel="noopener noreferrer" target="_blank">
-            <FacebookIcon className="footerIcon" />
-          </a>
+    <section
+      style={{
+        minHeight: '387px',
+        // TODO: need to update box shadow, current box shadow is a bit off from the design
```
Comment (DedsecKnight) [inline components/homeComponents/HomeFooter.tsx:12]: ^

**Review by DedsecKnight — COMMENTED** [review]
Just wondering if all of the `TODOs` needs to be addressed before merge, but everything looks good other than that.

File: components/homeComponents/HomeFooter.tsx (hunk 2 above)
Comment (uy-seng) [reply components/homeComponents/HomeFooter.tsx:12]: all TODO's are optional

**Review by DedsecKnight — APPROVED** [review]
LGTM :+1:

**Review by raywa04 — APPROVED** [review]
everything looks good. one optional thing ig for the footer, where it has the hack portal and non English stuff, keep the color of it the same for the mobile and laptop views.

**Pull request merged**